optimize.js

```diff
@@ -0,0 +1,79 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+async function viewDashboard([dashboardId]) {
+
+const accessToken = await getAccessToken();
```

Comment:
I would expect there to be some changes to `getAccessToken` to account for the fact that we will be using a different client ID and secret than we were when getting an access token for the Admin API. I don't know exactly what those changes would be, but I'd expect that the call to this function would require specification about which secret/key to use. Maybe we can pair on this to figure this out together?